### PR TITLE
fix docs describing order by clause in `SELECT`

### PR DESCRIPTION
**Pull request by stdrc**
## Description

We've supported `NULLS FIRST | LAST` since v0.18, this PR fixes the description of `ORDER BY` clause in `SELECT` command.

## Related code PR

https://github.com/risingwavelabs/risingwave/commit/41323a9eccadcb02f9bc471bfb07fb4c332d7448

## Checklist

- [x] I have checked the doc site preview, and the updated parts look good.
- [x] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

**Comment by stdrc** [comment]
* **#2659** <a href="https://app.graphite.dev/github/pr/risingwavelabs/risingwave-docs/2659?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stdrc and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by stdrc** [comment]
The changes should be applied for all versions on our doc site, can someone@doc-team help do the remaining job? Thanks very much! cc @hengm3467 @WanYixian

File: docs/sql/commands/sql-select.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Where `from_item` can be:
 |*alias*                    |A temporary alternative name for a table or materialized view in a query.|
 |*table_name*                    |A table or materialized view.|
 |*grouping_expression*      |<p>Values can be:</p><ul><li>Input column names</li><li>Input column expressions without subqueries or correlated columns</li></ul>|
-|**ORDER BY** clause        | The default sort order is **ASC**. Nulls options are not supported now. If the sort order is **ASC** or unspecified, nulls will be placed in front of non-null values. If the sort order is **DESC**, nulls will be placed after non-null values. This is different from the sort logic in PostgreSQL.|
+|**ORDER BY** clause        | The default sort order is **ASC**, and NULLs are considered largest. See [`ORDER BY` clause](../query-syntax/query-syntax-order-by-clause.md) for more details.|
```
Comment (hengm3467) [inline docs/sql/commands/sql-select.md]:
```suggestion
|**ORDER BY** clause        | By default, sorting is in ascending (ASC) order, with NULL values treated as the largest. For more information, see [`ORDER BY` clause](../query-syntax/query-syntax-order-by-clause.md).|
```

File: versioned_docs/version-2.0/sql/commands/sql-select.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ Where `from_item` can be:
 |*alias*                    |A temporary alternative name for a table or materialized view in a query.|
 |*table_name*                    |A table or materialized view.|
 |*grouping_expression*      |<p>Values can be:</p><ul><li>Input column names</li><li>Input column expressions without subqueries or correlated columns</li></ul>|
-|**ORDER BY** clause        | The default sort order is **ASC**. Nulls options are not supported now. If the sort order is **ASC** or unspecified, nulls will be placed in front of non-null values. If the sort order is **DESC**, nulls will be placed after non-null values. This is different from the sort logic in PostgreSQL.|
+|**ORDER BY** clause        | The default sort order is **ASC**, and NULLs are considered largest. See [`ORDER BY` clause](../query-syntax/query-syntax-order-by-clause.md) for more details.|
```
Comment (hengm3467) [inline versioned_docs/version-2.0/sql/commands/sql-select.md]:
```suggestion
|**ORDER BY** clause        | By default, sorting is in ascending (ASC) order, with NULL values treated as the largest. For more information, see [`ORDER BY` clause](../query-syntax/query-syntax-order-by-clause.md).|
```

**Review by hengm3467 — APPROVED** [review]
Added a couple of minor wording suggestions. Thanks so much for the prompt fix!

**Comment by hengm3467** [comment]
> The changes should be applied for all versions on our doc site, can someone@doc-team help do the remaining job? Thanks very much! cc @hengm3467 @WanYixian

@WanYixian Please help apply the changes to the remaining versions. Thanks a lot!

**Comment by WanYixian** [comment]
> > The changes should be applied for all versions on our doc site, can someone@doc-team help do the remaining job? Thanks very much! cc @hengm3467 @WanYixian
> 
> @WanYixian Please help apply the changes to the remaining versions. Thanks a lot!

Done.
1. Add `[ NULLS { FIRST | LAST } ]` into SQL SELECT since 0.18
2. Modify format in ORDER BY clause since [1.8](https://docs.risingwave.com/docs/1.8/query-syntax-order-by-clause/) (cuz this page exists since then)

**Review by WanYixian — APPROVED** [review]

**Pull request merged**